li-utils/src/main/pegasus/com/linkedin/common/ERModelRelationUrn.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.common
+
+/**
+ * Standardized ermodelrelation identifier.
+ */
+@java.class = "com.linkedin.common.urn.ERModelRelationUrn"
+@validate.`com.linkedin.common.validator.TypedUrnValidator` = {
+  "accessible" : true,
+  "owningTeam" : "urn:li:internalTeam:datahub",
+  "entityType" : "ermodelrelation",
+  "constructable" : true,
+  "namespace" : "li",
+  "name" : "ERModelRelation",
+  "doc" : "Standardized ERModelRelation identifier.",
+  "owners" : [ "urn:li:corpuser:fbar", "urn:li:corpuser:bfoo" ],
+  "fields" : [ {
+    "name" : "ermodelrelationId",
+    "doc" : "ERModelRelation native name e.g. <db>.<table>, /dir/subdir/<name>, or <name>",
+    "type" : "string",
+    "maxLength" : 284
+  }],
+  "maxLength" : 284
+}
+typeref ERModelRelationUrn = string
```

Comment:
let's rename to `ERModelRelationshipUrn`

are you sure we need a new urn type here?